storm-core/src/ui/public/index.html

```diff
@@ -143,6 +143,13 @@
             ]
           });
           $('#topology-summary [data-toggle="tooltip"]').tooltip();
+          $.getJSON("/api/v1/cluster/configuration", function(json){
+              var scheduler = json["storm.scheduler"];
+              if (scheduler != "backtype.storm.scheduler.resource.ResourceAwareScheduler"){
+                  $('#topology-summary td:nth-child(9),#topology-summary th:nth-child(9)').hide();
+                  $('#topology-summary td:nth-child(10),#topology-summary th:nth-child(10)').hide();
+              }
+          });
```

Comment:
I don't really like having this be configured based off of a hard coded scheduler.  I would prefer to have a separate config, or even better to have the rest API return a null for fields that it knows were not calculated properly.  Then have the template filter out those fields based off of those values.

If we are going to go with a javascript hide, please add a class to all of the fields you want to hide with something like RAS_ONLY and then just do ```$(".RAS_ONLY").hide();```